activerecord/test/cases/defaults_test.rb

```diff
@@ -83,6 +83,28 @@ def test_default_strings_containing_single_quotes
   end
 end
 
+if supports_text_column_with_default?
+  class DefaultTextTest < ActiveRecord::TestCase
+    class DefaultText < ActiveRecord::Base; end
+
+    setup do
+      @connection = ActiveRecord::Base.connection
+      @connection.create_table :default_texts do |t|
+        t.text :text_col, default: "Smith"
```

Comment:
Can you also add a test for quotes like here?

https://github.com/rails/rails/blob/9e9e11f05963ff5312a2fa1fa8bae5a6aa0e6dbe/activerecord/test/cases/defaults_test.rb#L67-L68

https://github.com/rails/rails/blob/9e9e11f05963ff5312a2fa1fa8bae5a6aa0e6dbe/activerecord/test/cases/defaults_test.rb#L77-L79

Reply:
Added